admin/install_server.sh

```diff
@@ -21,6 +21,11 @@ echo "client_encoding = utf8" >> "$PG_CONF"
 
 service postgresql restart
 
+# Less compiler
+curl -sL https://deb.nodesource.com/setup | sudo bash -
```

Comment:
I'm getting weird errors after installing less compiler using node and npm from the base image:

```
root@acousticbrainz-server:/vagrant/static/css# lessc
/usr/bin/env: node: No such file or directory
```

Installing node-less directly works and it compiles styling fine. I don't know if we'll need more recent version of nodejs in the future (for compiling JSX and other things).